### ISSUE TITLE
how to create new corpus

### ISSUE THREAD

**Issue by vstone386** [airbus-seclab/cpu_rec#6]
how to create new corpus from binaries or other files?

**Comment by LRGH** [comment]
Hello! It was documented, but not well documented...
I just have added a section in "How to use the tool" in the README.

**Comment by vstone386** [comment]
Thanks, I will try it!

**Comment by LRGH** [comment]
If it is not secret, can you tell which architecture you want to add to the corpus?

**Comment by vstone386** [comment]
I just want to make a experiment, if I create a larger corpus,dose the accuracy can be improved?

**Comment by LRGH** [comment]
The results of your experiment will be interesting.

My experiments showed me that it is very difficult to improve the accuracy of architecture detection when there are a lot of architectures in the corpus, especially because for many of them I had only very small samples. The easiest way to improve the accuracy of the tool is to remove the architectures that are unlikely to be used by the binary you are looking at.

I think that to improve the accuracy of the tool, one needs to use another technique than the one I have implemented. The problem is that if we want to use standard supervised machine learning techniques, we need a bigger corpus than what I have been able to collect.
